docs/configuration.md

```diff
@@ -48,6 +48,8 @@ Gohan server configuration uses YAML format.
   # list of etcd backend servers
   etcd:
       - "http://127.0.0.1:2379"
+  # timeout in milliseconds.1000 milliseconds by default 
```

Comment:
in line 51 you write that 1000 ms is the default
in line 52 you use 333 ms. It is misleading.